modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -21,6 +22,21 @@ about the update.
 
 Due to the infrequent use of our CSS modules, they will be going into maintenance mode in Canvas Kit v5. We will still support `@workday/canvas-kit-css` with bug fixes and significant visual updates, but it generally won't be receiving new components, additional features, etc. This will allow us to provide more focused support, and dedicate our efforts to making bigger and better improvements to our most used components: Canvas Kit React. If you have questions or concerns, please [let us know](https://github.com/Workday/canvas-kit/issues/new?labels=&template=question.md).
 
+### Codemod
```

Comment:
Codemod is now it's own section with code mod info specific to each breaking change.